x-pack/plugins/security_solution/server/lib/telemetry/sender.ts

```diff
@@ -445,11 +467,26 @@ export class TelemetryEventsSender {
           ...(licenseId ? { 'X-Elastic-License-ID': licenseId } : {}),
         },
       });
+      this.telemetryUsageCounter?.incrementCounter({
```

Comment:
You are also going to need to add these counters to the `sendOnDemand` function otherwise you won't get lists or endpoint telemetry.

Reply:
I will do so.

Reply:
Well, we'll get the counters incremented for `sendEvents` right? Just the queue-only counters won't get incremented. Both `processEvents` and `sendOnDemand` invoke `sendEvents`.

Reply:
So I don't think we're actually missing calls right?

Reply:
Ahh, my bad - this looks good.